packages/internals/src/engine-commands/getConfig.ts

```diff
@@ -60,8 +61,6 @@ type GetConfigErrorInit = {
 
 export class GetConfigError extends Error {
   constructor(params: GetConfigErrorInit) {
-    const headline = chalk.redBright.bold('Get Config: ')
-
```

Comment:
I don't remember that we decided to remove this part? 🤔

Reply:
I kind of agree that this is not helpful for users, it is _meta_ information about what happens behind the scenes. We should probably move it somewhere else (less prominent, end of line or similar) as context for us to understand what component triggered a warning or error.